src/Sylius/Bundle/CoreBundle/spec/Twig/ErrorRenderer/TwigErrorRendererSpec.php

```diff
@@ -0,0 +1,227 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace spec\Sylius\Bundle\CoreBundle\Twig\ErrorRenderer;
+
+use PhpSpec\ObjectBehavior;
+use Symfony\Bridge\Twig\ErrorRenderer\TwigErrorRenderer as DecoratedTwigErrorRenderer;
+use Symfony\Component\ErrorHandler\ErrorRenderer\ErrorRendererInterface;
+use Symfony\Component\ErrorHandler\Exception\FlattenException;
+use Symfony\Component\HttpFoundation\ParameterBag;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\HttpKernel\Exception\HttpException;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Twig\Environment;
+use Twig\Loader\LoaderInterface;
+
+final class TwigErrorRendererSpec extends ObjectBehavior
+{
+    function let(Environment $twig, RequestStack $requestStack, DecoratedTwigErrorRenderer $decoratedTwigErrorRenderer): void
+    {
+        $this->beConstructedWith($twig, $requestStack, $decoratedTwigErrorRenderer, false);
+    }
+
+    function it_implements_error_renderer_interface()
+    {
+        $this->shouldImplement(ErrorRendererInterface::class);
+    }
+
+    function it_renders_using_decorated_renderer_if_debug(
+        DecoratedTwigErrorRenderer $decoratedTwigErrorRenderer,
+        \Throwable $exception,
+        FlattenException $flattenException,
+        Environment $twig,
+        RequestStack $requestStack,
```

Comment:
```suggestion
        Environment $twig,
        RequestStack $requestStack,
        DecoratedTwigErrorRenderer $decoratedTwigErrorRenderer,
        \Throwable $exception,
        FlattenException $flattenException,
```
Sort of a soft convention to have things from `let` precede anything case-specific